libopae/plugins/xfpga/sysfs.c

```diff
@@ -777,13 +777,24 @@ fpga_result cat_handle_sysfs_path(char *dest, fpga_handle handle,
 	return cat_token_sysfs_path(dest, _handle->token, path);
 }
 
+STATIC char* cstr_dup(const char *str) {
+	size_t s = strlen(str);
```

Comment:
Might get dinging for not using strnlen_s. :(

Reply:
I think strlen is OK. We use it all over our codebase 😄